deploy/env-uzh-prd/values.yaml

```diff
@@ -0,0 +1,528 @@
+dfDomain: prd.df-app.ch
+
+heartbeats:
+  dailyGroupScores: https://betteruptime.com/api/v1/heartbeat/xN8ytcTixyVXsWnNjk9ndimD
```

Comment:
we should create new heartbeats in betterstack for the new domains (v3 deployment). ask RS for betterstack onboarding